oscrypto/keys.py

```diff
@@ -21,4 +27,100 @@
     'parse_pkcs12',
     'parse_private',
     'parse_public',
+    "make_pkcs12",
 ]
+
+
+def make_pkcs12(key, cert, other_certs, password):
+    # XXX check if key matches cert
```

Comment:
Here we need to add data type checks to make sure the parameters are of the correct type. I take the approach with oscrypto of not really supporting duck typing since this is a security context and I don't want the wrong types of things being passed and causing invalid or insecure results. You can look at some other methods for the functions to use and the format of the exceptions. The exception classes should be documented in the doc comments.